tests/lib/edge_impulse/src/edge_impulse_zip/edge-impulse-sdk/cmake/zephyr/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+set(EI_SDK_FOLDER ../../)
+
+include(${EI_SDK_FOLDER}/cmake/utils.cmake)
+
+target_include_directories(app PRIVATE
+    ${EI_SDK_FOLDER}
+    ${EI_SDK_FOLDER}/..
+)
+
+RECURSIVE_FIND_FILE(SOURCE_FILES "${EI_SDK_FOLDER}" "*.cpp")
+RECURSIVE_FIND_FILE(CC_FILES "${EI_SDK_FOLDER}" "*.cc")
+RECURSIVE_FIND_FILE(S_FILES "${EI_SDK_FOLDER}" "*.s")
+RECURSIVE_FIND_FILE(C_FILES "${EI_SDK_FOLDER}" "*.c")
```

Comment:
Please no.
Why this construct ?

Reply:
The code was actually using the same construct as CMakeLists.txt downloaded from Edge Impulse studio after deploying their model. I tried to keep it done in similar way.

After your comments, I will simplify the `CMakeLists.txt` tailoring it to our needs for the test.